packages/nextjs/src/client/index.ts

```diff
@@ -44,14 +44,17 @@ const globalWithInjectedValues = global as typeof global & {
 
 /** Inits the Sentry NextJS SDK on the browser with the React SDK. */
 export function init(options: BrowserOptions): void {
-  applyTunnelRouteOption(options);
-  buildMetadata(options, ['nextjs', 'react']);
+  const opts = {
+    environment: getVercelEnv(true) || process.env.NODE_ENV,
```

Comment:
Hey, @mydea !
Maybe I don't fully understand how it's supposed to work, but this breaks our app on update, cause we used to pass environment in options in sentry.client.config.ts, and now it seems to be ignored? It falls back to getVercelEnv in our case, which calls process, which doesn't exist in browser. I am not sure if it's a cause or maybe we're using it wrong and this code should never reach client?

We're getting Uncaught ReferenceError: process is not defined here:
<img width="697" alt="image" src="https://github.com/getsentry/sentry-javascript/assets/448437/00a658e0-202a-4d60-821a-d12fd841a87d">

Downgrading to 7.73.0, before this change, fixes it for us.
Any advise?

Reply:
Yeah, it does override it potentially, but it fails before cause `process` doesn't exist in client. I am on the go now, will try to create an issue later. Thank you!